src/test/ProtoGravaNFT.t.sol

```diff
@@ -112,7 +112,7 @@ contract ProtoGravNFTTestContract is ProtoGravaNFTTest {
     function testDefaultFormatSetAndGet() public {
         string memory newDefaultFormat = "retro";
         hevm.expectEmit(true, true, true, true);
-        emit Events.DefaultFormatChanged(newDefaultFormat);
+        emit protogravanft.DefaultFormatChanged(newDefaultFormat);
```

Comment:
Did you also try the contract name? i.e. `emit PhotoGravaNFT.DefaultFormatChanged`? I kind of assumed you could get to it via the "namespace" but maybe Solidity doesn't support it

Reply:
@holic thanks for the tip! i gave this a try (and updated PR) but still no dice

Reply:
😭 appreciate you giving it a whirl, the library approach is a good one too